napari-workshops/notebooks/manual_annotation.md

```diff
@@ -148,30 +153,38 @@ nbscreenshot(viewer)
 You can also get the number of cells of each class and an array of their centers as follows:
 
 ```{code-cell} ipython3
-print('Number of diving cells:', len(viewer.layers['dividing'].data))
-print('Number of non-diving cells:', len(viewer.layers['non-dividing'].data))
+print('Number of diving cells: ', len(viewer.layers['dividing'].data))
```

Comment:
Consider using f-strings.

`print(f"Number of diving cells: {len(viewer.layers['dividing'].data)}")`